src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="page"
+  lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/index.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body
+   class="page__body">
+    <div class="product-card">
+      <a
+        href="#"
+        class="product-card__link">
+        <img
+          src="./images/imac.jpeg"
+          alt="divice"
+          class="product-card__link__img"
+          >
+      </a>
+
+      <span class="product-card__name"
+      >APPLE A1419 iMac 27" Retina<br>5K Monoblock (MNED2UA/A)
+      </span>
```

Comment:
read the code style [rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) and make sure you write your code accordingly